litmus-portal/platforms/okteto/README.md

```diff
@@ -147,7 +147,7 @@ This directory contains setup guide to start developing Litmus Portal on Okteto
 
 - Finally click on `Deploy`
 
-- (Optional): For connecting to an external agent using the portal, annotate the server service and edit the environment variable for server's deployment as follows then restart the the server before using `litmusctl` to connect the agent.
+- (Optional): For connecting to an external agent using the portal, annotate the server service and edit the environment variable for server's deployment as follows restarting the server before using `litmusctl` to connect the agent.
```

Comment:
```suggestion
- (Optional): For connecting to an external agent using the portal, annotate the server service and edit the environment variable for server's deployment as follows and restart the server before using `litmusctl` to connect the agent.
```

Shouldn't it be like this? @ishangupta-ds

Reply:
Actually the container will auto restart with the pod once env var is edited as it's inside deployement's spec for the container. @Jonsy13 i guess the current change in this PR could better convey this, it looked like a manual restart earlier, which was not needed.

Reply:
Ok sure no issue